stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/messages/composer/MessageComposer.kt

```diff
@@ -60,20 +62,24 @@ public fun MessageComposer(
         DefaultComposerIntegrations(onAttachmentsClick)
     },
     label: @Composable () -> Unit = { DefaultComposerLabel() },
-    input: @Composable RowScope.() -> Unit = {
+    input: @Composable RowScope.(String, List<Attachment>, MessageAction?) -> Unit = { value, attachments, action ->
```

Comment:
Looks like a good direction to me 👍🏼

Reply:
Maybe we could make `MesasgeComposer` stateless too, extract `viewModel` param outside, and pass `State` type params instead? I think it's working nice in ui-components that we don't inject view models into ui components and we just bind view model with view inside Fragment, or Activity.